packages/apps/src/contexts/activity.ts

```diff
@@ -73,6 +73,24 @@ export interface IActivityContextOptions<T extends Activity = Activity> {
   next: (context?: IActivityContext) => (void | InvokeResponse) | Promise<void | InvokeResponse>;
 }
 
+interface ISignInOptions {
```

Comment:
Since this interface only has data members, why make it an interface.  For example, does this make testing easier in typescript?

Reply:
typescript doesn't have a concept of kwargs. So if you have many function args, and you don't want to supply some of them, then you have to do foo(undefined, undefined, "arg").

But using interfaces does:
foo({thirdArg: "arg"})